Custom client certificate (#2983)
mendix/docs: content/refguide/call-web-service-action.md

When you select **Override**, you can configure dynamically whether to use a proxy. You then supply the host, port, username, and password settings for the proxy.

### 4.8 Client certificate

> 

Reviewer: ```
{{% alert type="warning" %}}
This feature is available in Mendix 8.18.0 and later.
{{% /alert %}}
```

> In almost all cases, you can ignore this setting. **Use project settings** is a good default value.

Reviewer: ```suggestion
In most cases, the default **Use project settings** can be used.
```

> If desired, you can configure which client certificate to use for the request. These are the choices:

Reviewer: ```suggestion
However, you can specify a client certificate to use for the request by clicking **override**.
Select from one of the following:
```

> * **Use project settings** – use whatever settings are defined at the project level (default)

Reviewer: ```suggestion
* **Use project settings**(default) – use the settings that are defined at the project level 
```

* **Override** – override the project-level settings for this action

> When you select **Override**, you can configure which client certificate will be used. Specify the identifier for the client certificate. This identifier can be set in different places, depending on where you deploy the app:

Reviewer: ```suggestion
When you select **Override**, you can configure the client certificate that will be used. Click **Edit** to specify the **Client certificate identifier**. This identifier can be set in different places, depending on where you deploy the app:
```

> 1. When you deploy the app in the Mendix cloud, the identifier is set when [pinning a client certificate](https://docs.mendix.com/developerportal/deploy/certificates#3-outgoing-client-certificates). 

Reviewer: ```suggestion
* When you deploy the app to the Mendix cloud, the identifier is set when [pinning a client certificate](https://docs.mendix.com/developerportal/deploy/certificates#3-outgoing-client-certificates)
```

> 2. When you deploy the app elsewhere, the identifier is set in the custom setting [ClientCertificateUsages](custom-settings#ca-certificates)

Reviewer: ```suggestion
* When you deploy the app elsewhere, the identifier is set in the custom setting [ClientCertificateUsages](custom-settings#ca-certificates)
```
